src/client/client.go

```diff
@@ -129,8 +129,9 @@ func createRetry() func(response *resty.Response, err error) bool {
 		// On network errors - except hostname not found
 		if err != nil && (response == nil || response.StatusCode() == 0) {
 			switch {
-			case
-				strings.Contains(err.Error(), "No address associated with hostname"):
+			case strings.Contains(err.Error(), "No address associated with hostname"):
+				return false
+			case strings.Contains(err.Error(), "no such host"):
```

Comment:
Rovno s tym fixnem aj tuto drobnost.
Kazdy OS reportuje inak, ked je host nedostupny.
Tato hlaska `no such host` pokryva Windows a Alpine linux.

Narazil som na to, v jednom logu od CSAS ohladom ineho problemu.
Kedze su za firewallom, tak sa CLI snazi zbytocne pripojit `3 sekundy` k `api.github.com`:
![image](https://user-images.githubusercontent.com/19371734/132019087-bb5093c4-d71b-4413-85a2-845cd6e980fa.png)

Po tejto zmene (`return false`), sa nebude robit retry -> nema vyznam.